fe/fe-core/src/main/java/com/starrocks/planner/StreamLoadPlanner.java

```diff
@@ -246,19 +250,31 @@ public TExecPlanFragmentParams plan(TUniqueId loadId) throws UserException {
         // for stream load, we use exec_mem_limit to limit the memory usage of load channel.
         queryOptions.setMem_limit(streamLoadInfo.getExecMemLimit());
         queryOptions.setLoad_mem_limit(streamLoadInfo.getLoadMemLimit());
+
+        if (Config.enable_stream_load_profile) {
+            queryOptions.setEnable_profile(true);
+        }
+
         params.setQuery_options(queryOptions);
         TQueryGlobals queryGlobals = new TQueryGlobals();
         queryGlobals.setNow_string(DATE_FORMAT.format(new Date()));
         queryGlobals.setTimestamp_ms(new Date().getTime());
         queryGlobals.setTime_zone(streamLoadInfo.getTimezone());
         params.setQuery_globals(queryGlobals);
 
+        // Since stream load has only one fragment,
+        // the backend number can be directly assigned to 0
+        params.setBackend_num(0);
+        GlobalStateMgr.getCurrentState();
```

Comment:
meaningless here? nothing to do with the state

Reply:
ok